object_types/openshift/pod_log.yml

```diff
@@ -0,0 +1,77 @@
+version: 2016.06.21.0
+
+defaults:
+  type: string
+  doc_values: true
+
+pod_log:
+  type: group
+  description: >
+    Contains common fields, this is what populates the actual the mappings of the _default_.
+  fields:
+    - name: kubernetes
+      type: group
+      description: >
+        kubernetes-specific metadata.
+      fields:
+      - name: pod_name
+        type: string
+        index: not_analyzed
+        description: >
+          The name of the pod
+        norms:
+          enabled: True
+
+      - name: labels
+        type: group
+        description: >
+          Labels attached to the OpenShift object
+          Each label name is a subfield of labels field.
+          Each label name is de-dotted: dots in the name are replaced with
+          underscores.
+
+      - name: pod_id
+        type: string
+        index: not_analyzed
+        description: >
+          TODO
+        norms:
+          enabled: True
+
+      - name: namespace_name
+        type: string
+        index: not_analyzed
+        description: >
+          The name of the namespace
+        norms:
+          enabled: True
+
+      - name: namespace_id
+        type: string
+        index: not_analyzed
+        description: >
+          The name of the namespace
+        norms:
+          enabled: True
+
+      - name: hostname
```

Comment:
I would think it does not have to be `hostname`.  If we were deciding to document a mapping where the k8s `host` field was placed into these JSON documents as the top level `hostname` field then, then that change would be fine.

But here, were this namespace is for k8s, keeping names as close to what k8s already has seems like the best way to go.